gui/src/utils/hooks.ts

```diff
@@ -105,9 +105,10 @@ export const useFormatConfig = (): FormatConfig => {
                 timeZone: state.timeZone || TIMEZONE_CLIENT,
                 forceTZ: !!state.timeZone,
                 dateTime: state.dateTimeFormat || "yyyy-MM-dd HH:mm:ss zzz",
+                date: state.dateFormat || "yyyy/MM/dd",
```

Comment:
I suspect the proper format should be "yyyy-MM-dd" (homogeneous with the datetime format)